bundles/EcommerceFrameworkBundle/CartManager/CartItemInterface.php

```diff
@@ -27,10 +27,15 @@
 interface CartItemInterface
 {
     /**
-     * @return CheckoutableInterface
+     * @return CheckoutableInterface|null
      */
     public function getProduct();
 
+    /**
+     * @return int|null
+     */
+    public function getProductId();
```

Comment:
missing note in upgrade notes

Reply:
add it here: https://github.com/pimcore/pimcore/pull/9067/commits/054e8f6cbd1b3dca2c1103c0e7e6a13f3bd0a53b